index.html

```diff
@@ -5,43 +5,95 @@
+    <nav class="navbar navbar-expand-lg  bg-white border-secondary border mt-2 ">
+			<div class="container-fluid">
+				<a class="navbar-brand home" href="#">Awesome Books</a>
+				<button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarTogglerDemo03"
+					aria-controls="navbarTogglerDemo03" aria-expanded="false" aria-label="Toggle navigation">
+					<span class="navbar-toggler-icon"></span>
+				</button>
+				<div class="collapse navbar-collapse" id="navbarTogglerDemo03">
+					<ul class="navbar-nav ms-auto">
+						<li class="nav-item ">
+							<a class="nav-link list-button border-end" href="#">
+								Book List
+							</a>
+						</li>
+						<li class="nav-item">
+							<a class="nav-link add-new border-end" href="#">
+								New Book
+							</a>
+						</li>
+						<li class="nav-item add-contacts">
+							<a class="nav-link" href="#">
+								Contact
+							</a>
+						</li>
+					</ul>
+				</div>
+			</div>
+		</nav>
+    <div class="date "></div>
+    <div class="container  center rounded">
+      <section class="table-section">
+        <h2 class="center title">All The Books</h2>
+        <table class=" border center rounded">
+          <thead class="border">
+            <tr class="mt-1 border">
+              <th>Author</th>
+              <th>Title</th>
+              <th># of Pages</th>
+              <th>Delete</th>
+            </tr>
+          </thead>
+          <tbody id="book-list">
+      
+          </tbody>
+        </table>
+    </section>
+      <section class="form-section d-none">
+        <h1 class="center title">Awesome Books</h1>
+        <div class="border form-in center rounded">
+          <form id="book-form">
+            <div class="form-group">
+              <input placeholder="Title" type="text" id="title" class="form-control">
+            </div>
+            <div class="form-group">
+              <input placeholder="Author" type="text" id="author" class="form-control">
+            </div>
+            <div class="form-group ">
+              <input placeholder="# of Pages" type="text"  id="nPages" class="form-control ">
+            </div>
+              <input type="submit" value="Add Book" class="center btn " >
 
-        </form>
-      </div>
+          </form>
+        </div>
+    </section>
+      <section class="contact mx-5 mt-5 mb-5 d-none">
+        <h2 class="mb-5 text-center">Contact Information</h2>
+        <div class="center">
+          <p>Do Have any questions or want to say hello!</p>
+          <p>You can reach out to us</p>
+          <ul class=" mt-2">
+            <li>email: Contact@raquira.com</li>
+            <li>Phone: 123456789</li>
+            <li>Adress: center Raquira Boyaca</li>
+          </ul>
+        </div>
+      </section>
       </div>
+      <footer class="pt-2 fixed-bottom px-3 bg-light border-secondary border-top mt-5 text-center">
+          <p>&copy; Copyright 2021</p>

```

Comment:
- [ ] OPTIONAL
Again, as can be seen in the wireframe template, let's ensure that the `Copyright` is aligned to the left of the footer section